docs/spec_draft.md

```diff
@@ -1996,6 +1997,46 @@ where `predicate = rank(pred) == 0 ? pred : pred[i0, ..., iR-1]`.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.sign
+
+### Semantics
+
+Returns the sign of the `operand` element-wise and produces a `result` tensor.
+For each element `e`:
```

Comment:
can me make it `x`, e would appear as Euler's number in exponential op

Reply:
Also if we can refer something like IEEE-754 then we can remove this

Reply:
Or if we can deem this to be standard common knowledge / implementation everywhere (this I am not sure of)

Reply:
> can me make it x, e would appear as Euler's number in exponential op

Sure, will do.

> Also if we can refer something like IEEE-754 then we can remove this

No such operation exist in IEEE-754.

> Or if we can deem this to be standard common knowledge / implementation everywhere (this I am not sure of)

I'll keep this thread open for the next reviewer.